public/static/locales/en-US/account.json

```diff
@@ -27,7 +27,7 @@
       "email_helper": "Your school email ends with “sk.ac.th”.",
       "email_error": "Invalid email. Must end with “sk.ac.th”.",
       "password": "Password",
-      "password_helper": "Default is birthday, i.e. 832549 means Feb 8th 2006"
+      "password_helper": "Default is \"mysk2022autumnrelease\""
```

Comment:
I don’t think we should have this here. It’s probably best to let teachers inform students about this than to have it wide open to the public.

Also we use `“` and `“` not just `"`.